src/js/components/Header/VolumeControl.react.js

```diff
@@ -21,7 +34,7 @@ export default class VolumeControl extends Component {
 
         this.state = {
             showVolume : false,
-            volume     : Math.pow(audio.volume, 1 / 4),
+            volume     : unsmoothifyVolume(audio.volume, 1 / 4),
```

Comment:
I think this should be `unsmoothifyVolume(audio.volume, 4)`, since you're doing the `1 /` above